pkg/ccl/testccl/authccl/auth_test.go

```diff
@@ -240,6 +240,15 @@ func jwtRunTest(t *testing.T, insecure bool) {
 								t.Fatalf("wrong number of comma separated argumenets to jwt_cluster_setting ident_map: %d", len(a.Vals))
 							}
 							pgwire.ConnIdentityMapConf.Override(context.Background(), sv, strings.Join(args, "    "))
+						case "jwks_auto_fetch.enabled":
```

Comment:
I don't see this case added to any diffs in testdata. Am I missing something?

Reply:
The test cases related to this case has been added in pkg/ccl/jwtauthccl/authentication_jwt_test.go, not in the data driven. I added this case here for completeness and avoid missing definitions for data driven format.